src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -223,6 +223,28 @@ public Application.MemoryStatus MemoryStatus
         }
     }
 
+    /**
```

Comment:
please remove `@brief` and please use `//` for comments

Reply:
I see, I will update all of cases in this file.